medical_procedure_external/reports/medical_procedure_external_request_template_preview.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <template
+        id="medical_procedure_external_request_template_preview"
+        inherit_id="medical_procedure_external.medical_procedure_external_request_template_base"
+        primary="True"
+    >
+        <xpath
+            expr="//t[@t-call='medical_procedure_external.procedure_external_request_base_layout']"
+            position="attributes"
+        >
+            <attribute
+                name="t-call"
+            >medical_procedure_external.procedure_external_request_layout_preview</attribute>
+        </xpath>
+    </template>
+    <template
+        id="procedure_external_request_layout_preview"
+        inherit_id="medical_procedure_external.procedure_external_request_base_layout"
+        primary="True"
+    >
+        <xpath expr="//div[@name='patient_id']" position="inside">
+            <span>Template Patient Name</span>
+        </xpath>
+        <xpath expr="//div[@name='internal_identifier']" position="inside">
+            <span>MDR000000(0)</span>
```

Comment:
Not MDR, the same sequence that you define at ir_sequence_data.xml

Reply:
Thank you for your review! :smile: